verification/curator-service/ui/src/components/Users.tsx

```diff
@@ -85,113 +66,145 @@ class Users extends React.Component<Props, UsersState> {
+                            hidden: true,
+                        },
+                        {
+                            title: 'Name',
+                            field: 'name',
+                            type: 'string',
+                        },
+                        {
+                            title: 'Email',
+                            field: 'email',
+                            type: 'string',
+                        },
+                        {
+                            title: 'Roles',
+                            field: 'roles',
+                            type: 'string',
+                            render: (rowData): JSX.Element =>
+                                this.rolesFormControl(rowData),
+                        },
+                    ]}
+                    data={(query): Promise<QueryResult<TableRow>> =>
+                        new Promise((resolve, reject) => {
+                            let listUrl = this.state.url;
+                            listUrl += '?limit=' + query.pageSize;
+                            listUrl += '&page=' + (query.page + 1);
+                            this.setState({ error: '' });
+                            const response = axios.get<ListResponse>(listUrl);
+                            response
+                                .then((result) => {
+                                    const flattenedUsers: TableRow[] = [];
+                                    const users = result.data.users;
+                                    this.setState({ users: users });
+                                    for (const c of users) {
+                                        flattenedUsers.push({
+                                            id: c._id,
+                                            name: c.name || 'Name not provided',
+                                            email: c.email,
+                                            roles: c.roles,
+                                        });
+                                    }
+                                    resolve({
+                                        data: flattenedUsers,
+                                        page: query.page,
+                                        totalCount: result.data.total,
+                                    });
+                                })
+                                .catch((e) => {
+                                    this.setState({ error: JSON.stringify(e) });
+                                    reject(e);
+                                });
+                        })
+                    }
+                    title="Users"
+                    options={{
+                        search: false,
+                        filtering: false,
+                        sorting: false,
+                        padding: 'dense',
+                        draggable: false, // No need to be able to drag and drop headers.
+                        pageSize: 10,
```

Comment:
I had to make some changes to get choosing a pageSize to work in the other tables, could you add that here? https://github.com/open-covid-data/healthmap-gdo-temp/pull/548/files

Reply:
ah nice, thanks for pointing it out. Done.